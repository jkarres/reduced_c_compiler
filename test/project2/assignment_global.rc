// testing assignment of globals

int i;
float f;
bool b;

function : void main() {
  i = 10;
  cout << 1 << " " << i << endl;

  f = 3.1415;
  cout << 2 << " " << f << endl;

  b = true;
  cout << 3 << " " << b << endl;

}