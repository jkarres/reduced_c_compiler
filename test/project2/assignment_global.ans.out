1 10
2 3.14
3 true
